CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - [#34](https://github.com/SuperGoodSoft/solidus_taxjar/pull/34) Include API version in request headers
 - [#38](https://github.com/SuperGoodSoft/solidus_taxjar/pull/38) Added a rails engine to support future solidus backend UI
 - [#43](https://github.com/SuperGoodSoft/solidus_taxjar/pull/43) Support zeitwerk loading
+- [#47](https://github.com/SuperGoodSoft/solidus_taxjar/pull/47) Fixed bug in `validate_address_params` for addresses without a state
```

Comment:
Whoops I actually think we should make a new header above this, ## master, as 0.18.0 has already been released.

Reply:
Should everything else in here be in 0.18.0? I can start a new header!

Reply:
Yeah, we just cut a new release.

Reply:
This should be fixed now. I completely missed the change to the header in my rebase. Sorry about that.